src/lib.rs

```diff
@@ -105,7 +105,7 @@ fn field(s: &str) -> Tokens {
     } else {
         let snake = s.to_snake_case();
         if snake != s || snake.contains(|c: char| c == '$' || c == '#') {
-            let field = if snake == "$ref" {
+            let field = if snake == "ref" {
```

Comment:
This seems to have been accidentally included?

Reply:
This was intentional. The newer version of Inflector drops the `$`, so the comparison needed to be updated.